tests/test-team/config/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  testDir: '../',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
```

Comment:
do we want to allow retries in CI? i know we do this in Notify core but would it be good to start off here committing to writing non-flaky tests?

Reply:
updated